index.html

```diff
@@ -14,9 +14,50 @@
   </head>
   <body>
     <header>
-      
+      <a id="logo" href="/">My Logo</a>
+      <nav>
+        <span id="menu-btn">
+          <img src="assets/header/menu.svg" alt="menu button" />
+        </span>
+      </nav>
     </header>
-    <main></main>
+    <main>
+      <section id="headline">
+        <h1 id="primary-text">I'm Esteban. Glad to see you!</h1>
+        <p>
+          I’m a software developer! I can help you build a product , feature or
+          website Look through some of my work and experience! If you like what
+          you see and have a project you need coded, don’t hesitate to contact
+          me.
+        </p>
+        <p>LET'S CONNECT</p>
+        <ul class="socials">
+          <li id="twitter">
+            <a href="https://twitter.com/NaughTban"
+              ><img src="assets/headline/twitter.svg" alt="twitter icon"
+            /></a>
+          </li>
+          <li id="linkedin">
+            <a href="https://www.linkedin.com/in/esteban-palacios-5030a772/"
+              ><img src="assets/headline/linkedin.svg" alt="linkedin icon"
+            /></a>
+          </li>
+          <li id="medium">
+            <a href="/"
+              ><img src="assets/headline/medium.svg" alt="medium icon"
+            /></a>
+          </li>
+          <li id="github">
+            <a href="https://github.com/Estete9"
+              ><img src="assets/headline/github.svg" alt="github icon"
+            /></a>
+          </li>
+          <li id="angel-list">
+            <a href="/"><img src="assets/headline/angel list.svg" alt="" /></a>
+          </li>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Good job making all the changes mentioned by the previous reviewer. However, I think it would be better if you make all your social media icons to have the same size in order to stick as much ass possible to the Figma template design. Right now, they have different sizes, some are small and other are bigger.
![Screenshot from 2023-07-06 00-26-27](https://github.com/Estete9/Fullstack-Portfolio/assets/83042742/f65eaa63-bd22-4fa3-8d34-7b3be05a9362)